.github/BOTMETA.yml

```diff
@@ -240,7 +240,9 @@ files:
   $modules/network/nuage/: pdellaert
   $modules/network/nxos/: $team_nxos
   $modules/network/nso/: $team_nso
-  $modules/network/onyx/: $team_onyx
+  $modules/network/onyx/:
+    ignored: anasb
```

Comment:
is this necessary? or does removing the name do the trick?

Reply:
He is the author of one of the onyx-modules, so if he doesn't want any notifications, this is needed.

Reply:
Ok, so the author used the wrong Github handle for himself. Should be fixed now. cc @anasbadaha